packages/firestore/src/remote/existence_filter.ts

```diff
@@ -15,7 +15,9 @@
  * limitations under the License.
  */
 
+import { BloomFilter as ProtoBloomFilter } from '../protos/firestore_proto_api';
+
 export class ExistenceFilter {
   // TODO(b/33078163): just use simplest form of existence filter for now
-  constructor(public count: number) {}
+  constructor(public count: number, public unchangedNames?: ProtoBloomFilter) {}
```

Comment:
You can actually remove the comment on the line above this one. The linked bug is about adding bloom filters, which you are doing!